doc/source/whatsnew/v0.20.0.txt

```diff
@@ -29,8 +29,7 @@ New features
 
 Other enhancements
 ^^^^^^^^^^^^^^^^^^
-
-
+- Strings passed to ``DataFrame.groupby()`` as the ``by`` parameter may now reference either column names or index level names (:issue:`5677`)
```

Comment:
@jreback Example added here and in groupby.rst.  I didn't add anything to the groupby-docstring yet as I wasn't quite sure where it would fit in (there are only two examples there right now).  Let me know what you think.